extra_functions.sh

```diff
@@ -0,0 +1,16 @@
+# wheel renaming utility copied from github.com/MacPython/scipy-wheels
+
+function rename_wheel {
+    # Call with a name like scipy-1.5.0.dev0+58dbafa-cp37-cp37m-linux_x86_64.whl
```

Comment:
Can you give an example (in the comment) as to how the final name will look like? It seems that it will have a date after the build number.

Reply:
Sure, I just added an example.

(You can see final filenames from scipy here:  https://pypi.anaconda.org/scipy-wheels-nightly/simple/scipy/)